### DISCUSSION TITLE
joomla Update /Migration - List of conspicuous extensions and plugins with Extend function in Compatibility Checker

### DISCUSSION THREAD

**Discussion by WM-Loose** [joomla/joomla-cms#43389] [2. Cold Features]
Wouldn't it make sense for the upcoming migration to be able to deactivate or even uninstall the listed extensions and plugins, which could cause problems during a migration, either individually or all?

Simply asked:
A list is already displayed during the compatibility check, but it would be easier to be able to mark the listed extensions and plugins directly (single selection or select all) and then deactivate or optionally also uninstall directly on this page.

So far, you have to keep going back to the administration and then look for the listed extensions and plugins.

Regards,
Dirk

**Comment by crystalenka** [comment]
People going through an update process tend to do whatever just to get it done without necessarily thinking about the consequences; the friction here is a good thing because it encourages people to slow down and think about each of the extensions or plugins. 

So looking at this from a UX perspective, it would introduce a risk for people to lose data or make data inaccessible very easily without a way to revert. 

In the case of disabling: any data/functionality would not be accessible after disabling and if the extension is truly not compatible with the next version, it would not be possible to enable it again either. 

In the case of uninstalling: this removes everything associated with the extension which could impact places that the user might not expect (modules, content, plugins...) and the data would not be recoverable. 

Wanting to make things easier is good, but we have to proceed carefully when the risk is losing data or breaking the site. Some intentional friction is not a bad thing in these cases.
